locale/gu.yml

```diff
@@ -0,0 +1,107 @@
+False: "ખોટું"
+None: "કંઈ નહીં"
+True: "સાચું"
+and: "અને"
+as: "જેમ કે"
+assert: "દાવો"
+async: "async"
+await: "રાહ જોવી"
+break: "વિરામ"
+class: "વર્ગ"
+continue: "ચાલુ રાખો"
+def: "Def"
+del: "ડેલ"
+elif: "એલિફ"
+else: "બીજું"
+except: "સિવાય"
+finally: "છેવટે"
+for: "માટે"
+from: "માંથી"
+global:  વૈશ્વિક""
+if: "જો"
+import: "આયાત કરો"
+in: "માં"
+is: "છે"
+lambda: "લેમ્બડા"
+nonlocal: "નોનલોકલ"
+not: "નથી"
+or: "અથવા"
+pass: "પસાર"
+raise: "વધારો"
+return: "પાછા"
+try: "પ્રયાસ કરો"
+while: "જ્યારે"
+with: "સાથે"
+yield: "ઉપજ"
+
+# << Built-In Functions >>
+
+abs: "એબીએસ"
+all: "બધા"
+any: "કોઈપણ"
+ascii: "ascii"
+bin: "ડબ્બા"
+bool: "બુલ"
+breakpoint: "બ્રેકપોઇન્ટ"
+bytearray: "બાયટ્રે"
+bytes: "બાઇટ્સ"
+callable: "બોલાવવા યોગ્ય"
+chr: "chr"
+classmethod: "વર્ગમૂળ"
+compile: "સંકલન"
+complex: "જટિલ"
+delattr: "delattr"
+dict: "ડિક"
+dir: "dir"
+divmod: "divmod"
+enumerate: "ગણતરી કરવી"
+eval: "ઇવલ"
+exec: "એક્ઝેક"
+filter: "ફિલ્ટર"
+float: "ફ્લોટ"
+format: "બંધારણ"
+frozenset: "સ્થિર"
+getattr: "getattr"
+globals: "ગ્લોબલ્સ"
+hasattr: "ઉતાવળ કરવી"
+hash: "હેશ"
+help: "મદદ"
+hex: "હેક્સ"
+id: "આઈડી"
+input: "ઇનપુટ"
+int: "પૂર્ણાંક"
+isinstance: "isinstance"
+issubclass: "ઇસ્યુબક્લાસ"
+iter: "તે"
+len: "લેન"
+list: "યાદી"
+locals: "સ્થાનિકો"
+map: "નકશો"
+max: "મહત્તમ"
+memoryview: "મેમરીવ્યુ"
+min: "મિનિટ"
+next: "આગળ"
+object: "objectબ્જેક્ટ"
+oct: "ઓક્ટો"
+open: "ખુલ્લા"
+ord: "ઓર્ડર"
+pow: "પાવ"
+print: "છાપો"
+property: "મિલકત"
+range: "શ્રેણી"
+repr: "repr"
+reversed: ".લટું"
+round: "ગોળ"
+set: "સમૂહ"
+setattr: "setattr"
+slice: "કટકા"
+sorted: "સortedર્ટ થયેલ"
+staticmethod: "સ્થિર"
+str: "str"
+sum: "સરવાળો"
+super: "સુપર"
+tuple: "tuple"
+type: "પ્રકાર"
+vars: "યુદ્ધો"
+zip: "ઝિપ"
+__import__: "__ મહત્વપૂર્ણ__"
```

Comment:
This translation is great! Could you use the full [`language-keyword-template`](https://github.com/legesher/legesher-translations/blob/master/locale/language-keyword-template.yml) for when we start adding more programming language translations?